iphone/cli/commands/_build.js

```diff
@@ -3784,7 +3784,7 @@ iOSBuilder.prototype.writeInfoPlist = function writeInfoPlist() {
 	var defaultInfoPlistFile = path.join(this.platformPath, 'Info.plist'),
 		customInfoPlistFile = this.projectDir + '/Info.plist',
 		plist = this.infoPlist = new appc.plist(),
-		iphone = this.tiapp.iphone,
+		iphone = this.tiapp.iphone, // deprected in 2.1.0, removed in 7.0.0
```

Comment:
Since `iphone` is only used on like 3937 in the `if`, we could save a local variable and just do `if (this.tiapp.iphone) {`.